tests/integration/targets/aci_access_span_dst_group/tasks/main.yml

```diff
@@ -0,0 +1,323 @@
+    - change_ansible_group_1_again.current.0.spanDestGrp.attributes.name == "ansible_group_1"
+    - change_ansible_group_1_again.current.0.spanDestGrp.attributes.descr == "changed test span epg"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.attributes.name == "ansible_group_1"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.dscp == "VA"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.flowId == "2"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.ip == "10.0.0.2"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.mtu == "1500"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.srcIpPrefix == "10.0.2.2"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.tDn == "uni/tn-ansible_test/ap-ansible_test/epg-ansible_test"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.ttl == "4"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.ver == "ver1"
+    - change_ansible_group_1_again.current.0.spanDestGrp.children.0.spanDest.children.0.spanRsDestEpg.attributes.verEnforced == "yes"
+
+- name: Change span ansible_group path information (error)
+  cisco.aci.aci_access_span_dst_group:
+    <<: *add_ansible_group_1
+    destination_epg:
+      destination_ip: 10.0.0.2
+      source_ip: 10.0.2.2
+      tenant: ansible_test_2
+      ap: ansible_test_2
+      epg: ansible_test_2
+  register: error_on_path_change_ansible_group_1
+  ignore_errors: true
+
+- name: Verify change access span destination group type epg
+  ansible.builtin.assert:
+    that:
+    - error_on_path_change_ansible_group_1.msg == "APIC Error 105{{':'}} Span source or destination relation target configuration cannot be dynamically modified"
+
+- name: Add access span destination group type access interface port
+  cisco.aci.aci_access_span_dst_group: &add_ansible_group_2
+    <<: *aci_info
+    destination_group: ansible_group_2
+    description: test span access interface port
+    access_interface:
+      pod: 1
+      node: 101
+      path: eth1/1
+      mtu: 1500
+    state: present
+  register: add_ansible_group_2
+
+- name: Add access span destination group type access interface direct port channel
+  cisco.aci.aci_access_span_dst_group: &add_ansible_group_3
+    <<: *aci_info
+    destination_group: ansible_group_3
+    description: test span access interface port channel
+    access_interface:
+      pod: 1
+      node: 101
```

Comment:
Could not find the Node ID input box in the UI for Access Interface - Direct Port Channel. Is it creating the "Access Interface - Direct Port Channel" object?